templates/webapps/tool_shed/base_panels.mako

```diff
@@ -1,7 +1,7 @@
 <%inherit file="/base/base_panels.mako"/>
 
 ## Default title
-<%def name="title()">Galaxy Tool Shed</%def>
+<%def name="title()">Tool Shed</%def>
```

Comment:
@martenson I admittedly didn't test a local toolshed with this, but my read of it was that the "Galaxy" bit is preserved and that this doesn't change anything really.  Are you seeing otherwise?